src/image-player/image-player.tsx

```diff
@@ -28,6 +28,10 @@ export class ImagePlayer {
     this._preloadEnabled = preloadEnabled;
   }
 
+  public isImages() {
```

Comment:
method returns `number`. I would better make it returns `boolean` and rename to `isReady` or something similar